backend/src/main/resources/application.yml

```diff
@@ -0,0 +1,2 @@
+server:
```

Comment:
incl. a server port should not be necessary, also it will be overwritten when generating a Docker image as indicated in your Dockerfile

Reply:
thx. changeed.